locksmith/src/controllers/v2/priceController.ts

```diff
@@ -36,3 +38,30 @@ export const total: RequestHandler = async (request, response) => {
 
   return response.send(charge)
 }
+
+export const getCreditCardDetails: RequestHandler = async (
```

Comment:
I don't think this is the right name for the function. This does not give credit card details :)

Reply:
```suggestion
export const isCardPaymentEnabledForLock: RequestHandler = async (
```